public/bzflag/js/events.js

```diff
@@ -55,7 +55,9 @@ vwf_view.createdNode = function(nodeID, childID, childExtendsID, childImplements
         playerNode = childID;
         $('#createUser').dialog("open");
     }
-    if ( nodeID == vwf_view.kernel.findClients( "", "/" ) ) {
+    var appID = vwf_view.kernel.application();
+    var clients = vwf_view.kernel.find( appID, "doc('http://vwf.example.com/clients.vwf')/*" );
+    if ( clients && clients.length > 0 && ( nodeID == clients[ clients.length - 1 ] ) ) {
```

Comment:
This was an error in the original, wasn't it? `nodeID == vwf_view.kernel.findClients( "", "/" )` isn't meaningful. Since this section wasn't doing anything before, will it do the right thing now?